server/vulnerabilities/nvd/sync/testdata/cve/vulncheck_test_data/nvdcve-1.1-2024-expected.json

```diff
@@ -0,0 +1,243 @@
+        "affects": null,
+        "CVE_data_meta": {
+          "ASSIGNER": "psirt@paloaltonetworks.com",
+          "ID": "CVE-2024-0002"
+        },
+        "data_format": "MITRE",
+        "data_type": "CVE",
+        "data_version": "4.0",
+        "description": {
+          "description_data": [
+            {
+              "lang": "en",
+              "value": "A CVE Without Congiguration Nodes"
+            }
+          ]
+        },
+        "problemtype": {
+          "problemtype_data": []
+        },
+        "references": {
+          "reference_data": [
+            {
+              "name": "https://foo.com/CVE-2024-0002",
+              "url": "https://foo.com/CVE-2024-0002"
+            }
+          ]
+        }
+      },
+      "configurations": {
+        "CVE_data_version": "4.0",
+        "nodes": [
+          {
+            "cpe_match": [
+              {
+                "cpe23Uri": "cpe:2.3:a:foo:bar:2023.2.0.21408:*:*:*:*:*:*:*",
+                "vulnerable": true
+              }
+            ],
+            "operator": "OR"
+          }
+        ]
+      },
+      "impact": {},
+      "lastModifiedDate": "2024-02-15T06:23Z",
+      "publishedDate": "2024-02-14T18:15Z"
+    },
+    {
+      "cve": {
+        "affects": null,
+        "CVE_data_meta": {
+          "ASSIGNER": "zdi-disclosures@trendmicro.com",
+          "ID": "CVE-2024-0003"
+        },
+        "data_format": "MITRE",
+        "data_type": "CVE",
+        "data_version": "4.0",
+        "description": {
+          "description_data": [
+            {
+              "lang": "en",
+              "value": "A CVE that doesn't exist in the NVD Feed"
+            }
+          ]
+        },
+        "problemtype": {
+          "problemtype_data": []
+        },
+        "references": {
+          "reference_data": [
+            {
+              "name": "https://www.foo.com/support/security-bulletins.html",
+              "url": "https://www.foo.com/support/security-bulletins.html"
+            }
+          ]
+        }
+      },
+      "configurations": {
+        "CVE_data_version": "4.0",
+        "nodes": [
+          {
+            "cpe_match": [
+              {
+                "cpe23Uri": "cpe:2.3:a:foo:bar:2023.2.0.21408:*:*:*:*:*:*:*",
+                "vulnerable": true
+              }
+            ],
+            "operator": "OR"
+          }
+        ]
+      },
+      "impact": {},
+      "lastModifiedDate": "2024-04-03T17:24Z",
+      "publishedDate": "2024-04-03T17:15Z"
+    },
+    {
+      "cve": {
+        "affects": null,
+        "CVE_data_meta": {
+          "ASSIGNER": "zdi-disclosures@trendmicro.com",
+          "ID": "CVE-2024-0004"
+        },
+        "data_format": "MITRE",
+        "data_type": "CVE",
+        "data_version": "4.0",
+        "description": {
+          "description_data": [
+            {
+              "lang": "en",
+              "value": "A CVE without Configuration Nodes that doesn't exist in the NVD Feed"
```

Comment:
Nit: Curious to know if there's a good reason to include these (they won't match anything).

Reply:
I was on the fence with this, thought maybe it could be useful for troubleshooting.  It's easy enough to take out.

Reply:
Nah, let's leave it as is! (was just curious)